channels/microsoft365/Microsoft365errors.rst

```diff
@@ -0,0 +1,151 @@
+Microsoft 365 common errors

```

Comment:
We're already in the `Microsoft 365` section, maybe it's enough to name it `Common Errors`?